client/lib/domains/index.js

```diff
@@ -260,11 +261,22 @@ function getDomainProductSlug( domain ) {
 	return `dot${ tldSlug }_domain`;
 }
 
+function getDomainPrice( slug, productsList, currencyCode ) {
+	let price = get( productsList, [ slug, 'cost' ], null );
```

Comment:
This fails for me - it seems `cost` is a string. So the below line concatenates `9.00` with `13.00` and results in `9.0013.00` which is then formatted to `null`, because it's not a valid number.
(yes, I checked and have the newest code for this PR - and the backend is merged already, right?)